wgs.yaml

```diff
@@ -56,6 +56,67 @@ workinggroups:
   - name: katib
     owners:
     - https://raw.githubusercontent.com/kubeflow/katib/master/OWNERS
+- dir: wg-deployment
+  name: Deployment
+  mission_statement: >
+    WG Deployment covers developing, maintaining artifacts to deploy and operate Kubeflow.
+
+  charter_link: charter.md
+  label: area/wg-deployment
+  leadership:
+    chairs:
+    - github: Jeffwan
+      name: Jiaxin Shan
+      company: Amazon Web Services
```

Comment:
Call it AWS to be consistent with [Training WG](https://github.com/kubeflow/community/blob/master/wgs.yaml#L76)